packages/manager/modules/pci/src/projects/project/data-processing/submit-job/spark-sizing/spark-sizing.html

```diff
@@ -1,219 +1,237 @@
+            </oui-field>
+        </form>
+    </div>
+    <!--advanced config-->
+    <div data-ng-if="$ctrl.state.advancedSizing">
+        <form novalidate name="advancedSizingForm">
+            <div class="row">
+                <div class="col">
+                    <oui-field
+                        data-label="{{:: 'data_processing_submit_job_stepper_spark_driver_cores_label' | translate }}"
+                    >
+                        <div class="sizing__advanced-units-container">
+                            <oui-numeric
+                                name="driver-cores"
+                                data-model="$ctrl.state.driverCores"
+                                data-min="$ctrl.jobParameters.driver_cores.validator.min"
+                                data-max="$ctrl.jobParameters.driver_cores.validator.max"
+                            >
+                            </oui-numeric>
+                        </div>
+                        <span class="sizing__advanced-units">vCores</span>
+                    </oui-field>
+                </div>
+                <div class="col">
+                    <oui-field
+                        data-label="{{:: 'data_processing_submit_job_stepper_spark_driver_memory_label' | translate }}"
+                    >
+                        <div class="sizing__advanced-units-container">
+                            <oui-numeric
+                                name="driver-memory"
+                                data-model="$ctrl.state.driverMemoryGb"
+                                data-min="$ctrl.jobParameters.driver_memory.validator.min / 1024"
+                                data-max="$ctrl.maxMemoryDriverGb"
+                                data-on-change="$ctrl.onChangeDriverMemoryHandler(modelValue)"
+                            >
+                            </oui-numeric>
+                        </div>
+                        <span class="sizing__advanced-units">GiB</span>
+                    </oui-field>
+                </div>
+                <div class="col">
+                    <oui-field
+                        data-label="{{:: 'data_processing_submit_job_stepper_spark_driver_memory_overhead_label' | translate }}"
+                    >
+                        <div class="sizing__advanced-units-container">
+                            <oui-numeric
+                                name="driver-memory-overhead"
+                                data-model="$ctrl.state.driverMemoryOverheadMb"
+                                data-min="$ctrl.jobParameters.driver_memory_overhead.validator.min"
+                                data-max="$ctrl.maxOverheadMemoryDriverGb"
+                                data-disabled="$ctrl.state.advancedSizingDriverMemOverheadAuto"
+                            >
+                            </oui-numeric>
+                        </div>
+                        <span class="sizing__advanced-units">MiB</span>
```

Comment:
**question:** Dont we need translation for `MiB`?

Reply:
https://github.com/ovh/manager/pull/7067/commits/85a5cac61eebc3109c240870e7b9858a3345f8da